File: 27-RemoveElement/27-RemoveElement.py
# Last updated: 14/07/2026, 14:50:24
class Solution(object):
    def removeElement(self, nums, val):
       k=0
       for i in range(len(nums)):
        if(nums[i]!=val):
            nums[k]=nums[i]
            k=k+1
       return k
        

File: 27-RemoveElement/27-RemoveElement.java
// Last updated: 14/07/2026, 14:50:20
class Solution {
    public int removeElement(int[] nums, int val) {
        int p=0;
        for(int i=0;i<nums.length;i++){
            if(nums[i]!=val){
                nums[p]=nums[i];
                p++;
            }
        }
        return p;
    }
}